lockfile.json

```diff
@@ -24,7 +24,7 @@
     "revision": "14055bd46b383d18ebc9db219f8f6b048455ae04"
   },
   "elm": {
-    "revision": "b748941d615e1c4b84a0f8a1f3f3c07bc478c540"
+    "revision": "9c4987845ec240bb3b7fe72db05b42189b060802"
```

Comment:
this is already 0.19.0 .

Reply:
let's see if loading fails as it should

Reply:
I don't have any problems in using this grammar. Even if I generate all grammars from source with 0.19.0: Neovim reloads them without problems. I'm using a recent tree-sitter version but I though one before the ABI change.

Reply:
It works for some time... :smile: and crashes nvim. I only get a warning when I update the tree-sitter runtime.